src/components/Reports/PeopleReport/components/PeopleTasksPieChart.jsx

```diff
@@ -2,7 +2,7 @@ import { useSelector } from 'react-redux';
 import { PieChart } from '../../../common/PieChart';
 import { peopleTasksPieChartViewData } from '../selectors';
 import { ReportPage } from '../../sharedComponents/ReportPage';
-import { NewModal } from '../../../common/NewModal';
+import NewModal from '../../../common/NewModal';
```

Comment:
Typically a space is kept for destructured module imports, so it should be fine to keep it the original way:
import { Checkbox } from '../../common/Checkbox';

I would try installing ESLint and Prettier extensions in your IDE if you are using VS Code.